### PR TITLE
prepare for release 2.1

### PR DESCRIPTION
**Pull request by jclingan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <groupId>org.eclipse.microprofile</groupId>
     <artifactId>microprofile</artifactId>
     <packaging>pom</packaging>
-    <version>2.1-SNAPSHOT</version>
+    <version>2.1-RC1</version>
```
Comment (kwsutter) [inline pom.xml]: The Jenkins release builds take care of these type of changes.  Leave this with -SNAPSHOT.

File: spec/pom.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
     <parent>
         <groupId>org.eclipse.microprofile</groupId>
         <artifactId>microprofile</artifactId>
-        <version>2.1-SNAPSHOT</version>
+        <version>2.1-RC1</version>
     </parent>
```
Comment (kwsutter) [inline spec/pom.xml]: Same comment as earlier.  Leave this with -SNAPSHOT.

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 3]
```diff
@@ -30,6 +30,38 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile2.1]]
+=== MicroProfile 2.1 === (4Q2018)
+MicroProfile 2.1 is the eigth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+Based on MicroProfile's time-boxed release process, this is an incremental release that includes an update to OpenTracing 1.2.
+
```
Comment (kwsutter) [inline spec/src/main/asciidoc/architecture.asciidoc]: I would do a hotlink to OpenTracing 1.2, just to make it easy for readers to find the updated component.

Comment (kwsutter) [inline spec/src/main/asciidoc/architecture.asciidoc]: I would continue to reference the Java EE 8 vs Java EE 7 material (pointing them back at the 1.4 release).

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 4]
```diff
@@ -30,6 +30,38 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile2.1]]
+=== MicroProfile 2.1 === (4Q2018)
```
Comment (kwsutter) [inline spec/src/main/asciidoc/architecture.asciidoc]: Drop the === at the end of the section header.  Only the first === at the beginning are required.

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 5]
```diff
@@ -30,6 +30,38 @@ Only this Specification will be generated and maintained to document the content
...
+----
+<dependency>
+    <groupId>org.eclipse.microprofile</groupId>
+    <artifactId>microprofile</artifactId>
+    <version>2.1</version>
+    <type>pom</type>
+    <scope>provided</scope>
+</dependency>
+----
+
```
Comment (kwsutter) [inline spec/src/main/asciidoc/architecture.asciidoc]: You accidentally dropped the pointer to our github repo:  "Here is the link to the github repository for this Eclipse-based project."

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 3 above)
Comment (jclingan) [reply spec/src/main/asciidoc/architecture.asciidoc]: Resolved.

Comment (jclingan) [reply spec/src/main/asciidoc/architecture.asciidoc]: Resolved.

**Pull request merged**